core-bundle/src/String/HtmlAttributes.php

```diff
@@ -419,34 +418,30 @@ private function escapeValue(string $name, string $value): string
     private function parseStyles(string $styles): array
     {
         // Regular expression to match declarations according to https://www.w3.org/TR/css-syntax-3/#declaration-list-diagram
-        $declarationRegex = <<<'EOD'
-            (
-                (?:
-                    \\.                           # Escape
-                    |"(?:\\.|[^"\n])*+(?:"|\n|$)  # String token double quotes
-                    |'(?:\\.|[^'\n])*+(?:'|\n|$)  # String token single quotes
-                    |\{(?:(?R)|[^}])*+(?:\}|$)    # {}-block
-                    |\[(?:(?R)|[^]])*+(?:\]|$)    # []-block
-                    |\((?:(?R)|[^\)])*+(?:\)|$)   # ()-block
-                    |[^;{}\[\]()"']               # Anything else
-                )++
-            )ixs
-            EOD;
+        $declarationRegex = '(
+            (?:
+                \.                              # Escape
+                |"(?:\.|[^"\n])*+(?:"|\n|$)     # String token double quotes
+                |\'(?:\.|[^\'\n])*+(?:\'|\n|$)  # String token single quotes
```

Comment:
@ausi Is it correct to shorten `\\.` to `\.` here?

Reply:
```suggestion
                \.                              # Escape
                |"(?:\\\.|[^"\n])*+(?:"|\n|$)     # String token double quotes
                |\'(?:\\\.|[^\'\n])*+(?:\'|\n|$)  # String token single quotes
```
No, I think they have to be extendet to `\\\.` or `\\\\.`.
Did the unit tests not fail with `\.`? If so, I need to look into it.

Reply:
> If so, I need to look into it.

Done, added test in 1133ea0df49c22ba126feb34c3c68e2f293e8075